2023-02-22 05:44:52,216 [MainThread  ] [INFO ] [root]  STARTING BROWSER: /home/jake/research/chromium/src/out/baseline-no-gc/chrome
2023-02-22 05:44:52,216 [MainThread  ] [INFO ] [root]  STARTING BROWSER: driver: /home/jake/research/chromium/src/out/baseline-no-gc/chromedriver
2023-02-22 05:44:52,217 [MainThread  ] [INFO ] [root]  STARTING BROWSER: args: --no-default-browser-check --disable-component-update --disable-sync --no-experiments --enable-benchmarking --disable-extensions --no-first-run --disable-background-timer-throttling --disable-renderer-backgrounding --window-size=1500,1000 --enable-logging --log-file=/home/jake/research/v8_handles_experiments/crossbench/chrome_v112_chrome-handles-no-gc_default/speedometer_2.1/6/browser.chrome.log
2023-02-22 05:44:55,522 [MainThread  ] [INFO ] [root]  RUNNING STORY
2023-02-22 05:46:48,127 [MainThread  ] [WARNI] [urllib3.connectionpool]  Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbcaa363a90>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/9ca075663b264c798f81f7f9f592cd65
2023-02-22 05:46:48,128 [MainThread  ] [WARNI] [urllib3.connectionpool]  Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbcaa363df0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/9ca075663b264c798f81f7f9f592cd65
2023-02-22 05:46:48,128 [MainThread  ] [WARNI] [urllib3.connectionpool]  Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbcaa3588b0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/9ca075663b264c798f81f7f9f592cd65
